AdyenActions/Components/QRCode/QRCodeActionComponent.swift

```diff
@@ -132,9 +144,16 @@ public final class QRCodeActionComponent: ActionComponent, Cancellable {
     private func updateExpiration(_ timeLeft: TimeInterval) {
         progress.completedUnitCount = Int64(timeLeft)
         let timeLeftString = timeLeft.adyen.timeLeftString() ?? ""
-        expirationText = localizedString(.pixExpirationLabel,
-                                         configuration.localizationParameters,
-                                         timeLeftString)
+        guard let action = qrCodeAction else {return}
+        if action.paymentMethodType == .promptPay {
```

Comment:
Could we use a switch for this type? I think there are more qrCode types on the way so let's be error-proof in advance :)